From 89bb04a01a2d0b71c56c918961f32fe8e80d8234 Mon Sep 17 00:00:00 2001
From: Andrew Gregory <andrew.gregory.8@gmail.com>
Date: Mon, 11 Feb 2013 22:13:30 -0500
Subject: [PATCH] do not use SYSCONFDIR for binfmt.d

This makes binfmt.d consistent with tmpfiles.d and systemd which uses
hard coded paths for both.

Signed-off-by: Andrew Gregory <andrew.gregory.8@gmail.com>
---
 init.d/procfs.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.d/procfs.in b/init.d/procfs.in
index 992567e..7b7c6e6 100644
--- a/init.d/procfs.in
+++ b/init.d/procfs.in
@@ -31,7 +31,7 @@ start()
 				ebegin "Loading custom binary format handlers"
 				fmts=$(grep -hsv -e '^[#;]' -e '^[[:space:]]*$' \
 					/run/binfmt.d/*.conf \
-					@SYSCONFDIR@/binfmt.d/*.conf \
+					/etc/binfmt.d/*.conf \
 					""/usr/lib/binfmt.d/*.conf)
 				if [ -n "${fmts}" ]; then
 					echo "${fmts}" > /proc/sys/fs/binfmt_misc/register
-- 
1.8.3

